lib/perl/Genome/InstrumentData/Command/Import/Inputs.pm

```diff
@@ -0,0 +1,110 @@
+package Genome::InstrumentData::Command::Import::Inputs;
+
+use strict;
+use warnings;
+
+use Genome;
+
+use Genome::InstrumentData::Command::Import::Inputs::SourceFiles;
+
+class Genome::InstrumentData::Command::Import::Inputs {
+    is => 'UR::Object',
+    id_by => [ # use arrayref to keep order
+        process_id => { is => 'Text', },
+        line_number => { is => 'Number', },
+    ],
+    has => {
+        entity_params => {
+            is => 'HASH',
+            default_value => {
+                individual => {}, sample => {}, library => {},
+            },
+        },
+        source_paths => { is => 'Text', is_many => 1, },
+    },
+    has_optional => {
+        analysis_project_id => { is => 'Text', },
+    },
+    has_transient => {
+        format => { via => 'source_files', to => 'format', },
+        library_name => { via => 'library', to => 'name', },
+        sample_name => { via => 'library', to => 'sample_name', },
+    },
+};
+
+sub get { shift; Genome::InstrumentData::Command::Import::Inputs::Factory->get->from_inputs_id(@_); }
+sub create { Carp::confess('Use inputs factory to create!'); }
+
+sub lib_and_source_file_md5sum {
+    my $self = shift;
+    return substr(
+        Genome::Sys->md5sum_data( join(' ', $self->library_name, $self->source_paths) ), 
+        0, 6,
+    );
+}
+
+sub analysis_project {
+    my $self = shift;
+    if ( $self->process ) {
```

Comment:
@tmooney  Currently, there are 2 entry points to import instdata. Manual (or single), by invoking the 'basic' command. and one a little more automated (multiple) through the 'launch' command. The basic command does not create a process. And a 'fake' process id needs to be used. I hope to eliminate the basic importer to have just one import command. As a tie over, a process could be created in the basic command. Since I hope to eliminate it, I did not add that bit of code. Do you have an opinion?